### ISSUE TITLE
[Question] xcape send multiple keys 

### ISSUE THREAD

**Issue by azzamsa** [alols/xcape#105]
I tried to map `return` to `control` using this

``` bash
xmodmap -e "remove Control = Control_R"
xmodmap -e "keycode 0x69 = Return"
xmodmap -e "keycode 0x24 = Control_R"
xmodmap -e "add Control = Control_R"

xcape -t 10000 -e "Control_R=Return"
```
When I hit `return` I get multiple `return` sent to my application. And when I hold `Ctrl_L` I get multiple `return` too, not `Ctrl` it self.
Is this normal ?

**Comment by brotherJ4mes** [comment]
Not sure if this is the case for you but when I was experimenting with xcape I learned that calls to it are cumulative (i.e. if you run your xcape command twice, then you'll get two ```return```'s when you press ```Control```.)  A simple ```killall xcape``` would reset xcape and would fix this issue for me.

**Comment by azzamsa** [comment]
Whoa, I've dropped xcape for long time. About a year now. But today I got new laptop and want to give it a second try.

@brotherJ4mes after reading your comments, I realize that my .bash_profile little messed up (I believe it reloaded more than one). So I just put `xcape -t 10000 -e "Control_R=Return"` in my i3 config. And wow, it works great.

Now I can use return key for 'return', and for 'ctrl' if I hold it. what a great app. 

Thanks again @brotherJ4mes for giving me a clue.
